src/axom/sidre/core/MFEMSidreDataCollection.cpp

```diff
@@ -2419,15 +2419,15 @@ void MFEMSidreDataCollection::reconstructFields()
       {
         // The vdim is being overwritten here with the value in the basis string so we
         // can correctly construct the QuadratureFunction
-        m_quadspaces.emplace(basis_name,
-                             detail::NewQuadratureSpace(basis_name, mesh, vdim));
+        m_quadspaces[basis_name] = std::unique_ptr<mfem::QuadratureSpace>(
```

Comment:
@joshessman-llnl -- this is the issue that I was seeing that prevented me from upgrading to the `MFEMSidreDataCollection`.  Once this is in, I think I'll be able to upgrade.

Reply:
Can we flip the `AXOM_ENABLE_MFEM_SIDREDATACOLL` default then (or just remove the conditional entirely)? Or should that wait on a confirmed upgrade?

Reply:
I'm ok with flipping the default (in a separate PR). 

I don't think we should remove this option until we have a confirmed update.